### PR TITLE
Added AktualizrConfigChanged

### PR DESCRIPTION
**Pull request by jochenschneider**
Signed-off-by: Jochen Schneider <j.schneider@here.com>

### CODE REVIEW

File: libats-messaging-datatype/src/main/scala/com/advancedtelematic/libats/messaging_datatype/Messages.scala [hunk 1]
```diff
@@ -168,6 +168,9 @@ object Messages {
 
   final case class CommitManifestUpdated(namespace: Namespace, commit: Commit, releaseBranch: String, metaUpdaterVersion: String)
 
+  final case class AktualizrConfigChanged(namespace: Namespace, uuid: DeviceId, pollingSec: Int,
```
Comment (simao) [inline libats-messaging-datatype/src/main/scala/com/advancedtelematic/libats/messaging_datatype/Messages.scala]: Lets include some kind of timestamp when it was received

Comment (jochenschneider) [reply libats-messaging-datatype/src/main/scala/com/advancedtelematic/libats/messaging_datatype/Messages.scala]: Ok. Same for CommitManifestUpdated?

Comment (simao) [reply libats-messaging-datatype/src/main/scala/com/advancedtelematic/libats/messaging_datatype/Messages.scala]: @jochenschneider yes that would be great

**Comment by jochenschneider** [comment]
I had thought the migration would be ok by using a default value, but that only seems to be taken into account by circe if you use a different configuration from circe-extras. Not great, I don't know if there is another generic solution.

**Comment by simao** [comment]
It would work with Option[Instant], but it'd rather just do the migration slowly and should be fine.

**Comment by jochenschneider** [comment]
I think I've added optional timestamps in the past, but I agree.

**Pull request merged**